src/components/BoardHeader/BoardHeader.scss

```diff
@@ -42,30 +42,61 @@ $board-header__hover-padding: $board-header__hover-padding-top-bottom $padding--
 
 .board-header__users {
   position: absolute;
-  right: 0;
+  right: $margin--large;
   top: calc(24px - #{$board-header__hover-padding-top-bottom});
-  border-radius: 16px;
-  padding: $board-header__hover-padding;
+  padding: $board-header__hover-padding-top-bottom $padding--small;
 
   display: flex;
-  flex-direction: row-reverse;
+  flex-direction: row;
+  align-items: center;
 
   background: none;
   border: none;
   outline: none;
+}
+
+.board-header__share-button {
+  color: $color-backlog-blue;
+  outline: none;
+  border: 2px solid $color-backlog-blue;
+  border-radius: 100%;
+  height: 41.14px;
+  width: 41.14px;
+  margin-top: 3px;
```

Comment:
There are probably better options than to use magic numbers this e. g. using flex-end instead of center